Mathlib/Topology/Order/LowerUpperTopology.lean

```diff
@@ -20,20 +20,20 @@ right-closed left-infinite intervals.
 
 ## Main statements
 
-- `LowerTopology.t0Space` - the lower topology on a partial order is T₀
-- `LowerTopology.isTopologicalBasis` - the complements of the upper closures of finite
+- `IsLower.t0Space` - the lower topology on a partial order is T₀
+- `IsLower.isTopologicalBasis` - the complements of the upper closures of finite
```

Comment:
```suggestion
- `Topology.IsLower.t0Space` - the lower topology on a partial order is T₀
- `Topology.IsLower.isTopologicalBasis` - the complements of the upper closures of finite
```

Reply:
I think that's what I had originally https://github.com/leanprover-community/mathlib4/pull/7310/commits/c3c74570a0dac68377b95f55e40a66a068e08a1a#diff-00624ef46078276636f22330988bc2becf8bef0223f4560eb2d00b4e19e54386R23 but @YaelDillies changed it back:

https://github.com/leanprover-community/mathlib4/pull/7310/commits/6ec78992b4f30a3441a210a37139c8409fe5fde7#diff-00624ef46078276636f22330988bc2becf8bef0223f4560eb2d00b4e19e54386R23

Reply:
Ah right, I think `doc-gen` is now a bit better at linking not-fully-qualified names, so you can keep it like that. You can ignore these comments then.